types/ember-data/index.d.ts

```diff
@@ -35,11 +35,18 @@ interface AttributeMeta<Model extends DS.Model> {
     parentType: Model;
     isAttribute: true;
 }
+
+interface RelationshipMetaOptions {
+    async?: boolean;
+    inverse?: string;
+    polymorphic?: boolean;
```

Comment:
One of the requirements of these types is that they describe only the library's public API. 

Looking through the docs for examples of this `RelationshipMeta` object, I can find [references to `async` and `inverse`](https://api.emberjs.com/ember-data/3.13/functions/@ember-data%2Fmodel/hasMany) but can't find anything that declares `polymorphic` to be a public thing. 

Do you have some basis for believing `polymorphic` is a public thing?